src/libs/PolicyUtils.ts

```diff
@@ -609,11 +609,11 @@ function getManagerAccountID(policy: OnyxEntry<Policy> | SearchPolicy, expenseRe
     }
 
     const employee = policy?.employeeList?.[employeeLogin];
-    if (!employee) {
+    if (!employee && !defaultApprover) {
```

Comment:
@JKobrynski I think this is a good candidate for a unit test.

Reply:
All right, I'm on it!